src/Parser/Paladin/Protection/Modules/Items/TyelcaFerrenMarcussStature.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import Wrapper from 'common/Wrapper';
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import calculateEffectiveDamage from 'Parser/Core/calculateEffectiveDamage';
+import calculateEffectiveHealing from 'Parser/Core/calculateEffectiveHealing';
+import ItemDamageDone from 'Main/ItemDamageDone';
+import ItemHealingDone from 'Main/ItemHealingDone';
+
+/*
+* Equip: Avenger's Shield deals 20% increased damage and jumps to 2 additional targets.
+*/
+
+const debug = false;
+
+const TYELCA_MODIFIER = 0.2;
+
+class TyelcaFerrenMarcussStature extends Analyzer {
+	static dependencies = {
+		combatants: Combatants,
+	}
+
+	damageDone = 0;
+	healingDone = 0;
+
+	on_initialized() {
+		this.active = this.combatants.selected.hasLegs(ITEMS.TYELCA_FERREN_MARCUSS_STATURE.id);
+	}
+
+	on_byPlayer_cast(event) {
+		const spellId = event.ability.guid;
+		if (spellId !== SPELLS.AVENGERS_SHIELD.id) {
+			return;
+		}
+		this.targetsHit = 0;
+	}
+
+	on_byPlayer_damage(event) {
+		const spellId = event.ability.guid;
+		if (spellId !== SPELLS.AVENGERS_SHIELD.id) {
+			return;
+		}
+		if(this.targetsHit > 5) {
+			debug && console.log('this is a bug');
+		}
+		//If you hit a 4th or 5th target because of the pants all that damage should be attributed to them
+		if(this.targetsHit > 3) {
+			this.damageDone = (event.amount || 0) + (event.absorbed || 0);
+			this.targetsHit++;
+		}
+		else {
+			this.damageDone += calculateEffectiveDamage(event, TYELCA_MODIFIER);
+			this.targetsHit++;
+		}
+	}
+
+	on_byPlayer_absorbed(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.BULWARK_OF_ORDER_SHIELD.id) {
+      return;
+    }
+    this.healingDone += calculateEffectiveHealing(event, TYELCA_MODIFIER);
```

Comment:
I looked into implementing this and only the first applies the Bulwark of Order buff. I can calculate how much shield they gained from the extra hits on the damage event but then there is no way to tell if that extra amount was actually absorbed. I am going to use the `approximate` prop and say that if you hit more than 3 targets this may be higher.